README.md

```diff
@@ -239,3 +239,18 @@ Once application is installed create connection to DataSet environment under `Co
 Note that build cleans previously created configuration. To prevent removal of configuration while build 
 - copy configured configuration to home folder `mkdir -p ~/splunk_dataset_app_configuration && cp -R ./output/TA_dataset/local/* ~/splunk_dataset_app_configuration/`
 - copy back to splunk `mkdir -p ./output/TA_dataset/local/ && cp -R ~/splunk_dataset_app_configuration/* ./output/TA_dataset/local/`
+
+
+## My Development Workflow
```

Comment:
This repo is public, I would not document anything `My ....` rather `Alternative .....`
```suggestion
## Alternative Build and Run Workflow for Docker
```